### ISSUE TITLE
some abc tunes are too wide for the tune page

### ISSUE THREAD

**Issue by tcizadlo** [slow-session/slow-session.github.io#11]
some abc tunes are a little too wide to fit in the window provided.  The whole tune can be viewed, but you have to scroll.  Resizing the whole browser does not help.  Could the tunes be scaled to fit?

**Comment by edwardabraham** [comment]
Yes indeed, the layout can be improved. Firstly, the  layout of the tunes
reflects the line-breaks in the ABC. So, you can put fewer bars on a line
to get a better layout. Secondly, we can scale the display to fit the
window width.

**Comment by tcizadlo** [comment]
Did a work around in the ABC -> .md file generator reformatting ill formatted ABCs
